corehq/apps/reports/sqlreport.py

```diff
@@ -285,6 +285,19 @@ def _get_data(self, start=None, limit=None):
     def data(self):
         return self._get_data()
 
+    def get_sql_queries(self):
+        qc = self.query_context()
+        for c in self.columns:
+            qc.append_column(c.view)
+
+        session_helper = connection_manager.get_session_helper(self.engine_id)
+        with session_helper.session_context() as session:
+            return qc.get_query_strings(session.connection())
+
+    def print_sql_queries(self):
+        for query in self.get_sql_queries():
+            print(query)
```

Comment:
Is this before or after the params get resolved? meaning will this display `WHERE age_in_months = :age_0` or `WHERE age_in_months = 0` (just for curiosity either is fine for me)